add-to-pydotorg.py

```diff
@@ -242,7 +242,7 @@ def has_sigstore_signature(rfile):
     ]
 
     print('Signging release files with Sigstore')
-    run_cmd(['python3', '-m', 'sigstore', 'sign', '--oidc-disable-ambient-providers'] + filenames)
+    run_cmd(['python3', '-m', 'sigstore', 'sign', '--overwrite', '--oidc-disable-ambient-providers'] + filenames)
```

Comment:
This is the wrong fix for that. I made it better in 21e1adaf9b04ffbcf8d3699bf11f1ecac3db239c